iphone/cli/commands/_build.js

```diff
@@ -2956,6 +2956,8 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 				obj.path = xobjs.PBXFileReference[id + '_comment'] = '"' + appName + '.entitlements"';
 			} else if (obj.path === 'Titanium.app') {
 				obj.path = xobjs.PBXFileReference[id + '_comment'] = '"' + appName + '.app"';
+			}	else if (obj.path === '"Titanium-Bridging-Header.h"') {
+				obj.path = xobjs.PBXFileReference[id + '_comment'] = '"' + scrubbedAppName + '-Bridging-Header.h"';
```

Comment:
I find the ES6+ style string concat a little easier on the eyes:
```suggestion
				obj.path = xobjs.PBXFileReference[id + '_comment'] = `"${scrubbedAppName}-Bridging-Header.h"`;
```

Reply:
Updated.